tools/backlog-groomer/tests/test_config.py

```
@@ -0,0 +1,164 @@
+def test_bad_cadence_raises(tmp_path):
+    with pytest.raises(ValueError, match="'cadence' must be a preset name"):
+        config.load(write(tmp_path, "cadence: whenever\n"))
+
+
+def test_cadence_accepts_preset_and_raw_cron(tmp_path):
+    assert config.load(write(tmp_path, "cadence: daily\n")).cadence == "daily"
+    assert config.load(write(tmp_path, "cadence: 41 5 * * *\n")).cadence == "41 5 * * *"
+
+
+# ---------------------------------------------------------------------------
+# armed(): the full 2×2 matrix (AC3), plus the strictness edges
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)